proto2ros/CMakeLists.txt

```diff
@@ -13,19 +13,47 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+
+find_package(Protobuf REQUIRED)
 
 rosidl_generate_interfaces(${PROJECT_NAME}
   msg/Any.msg msg/AnyProto.msg msg/Bytes.msg msg/List.msg
   msg/Struct.msg msg/StructEntry.msg msg/Value.msg
 )
 
+add_library(${PROJECT_NAME}_conversions SHARED src/conversions.cpp)
+target_include_directories(${PROJECT_NAME}_conversions PUBLIC
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
+)
```

Comment:
```
target_compile_options(${PROJECT_NAME}_conversions
    PUBLIC
    "$<$<CONFIG:DEBUG>:-O0>"
    "$<$<OR:$<CONFIG:RELWITHDEBINFO>,$<CONFIG:RELEASE>>:-O3>"
    "$<$<OR:$<CONFIG:RELWITHDEBINFO>,$<CONFIG:DEBUG>>:-g>")
target_compile_definitions(${PROJECT_NAME}_conversions
    PUBLIC
    "$<$<CONFIG:DEBUG>:DEBUG>"
    "$<$<OR:$<CONFIG:RELWITHDEBINFO>,$<CONFIG:RELEASE>>:NDEBUG>")

Reply:
Hmm, do we have to? CMake already bundles and selects [per-compiler configuration](https://github.com/Kitware/CMake/tree/d6e03018cb67698366af0742cfaa49ec8c29d0fb/Modules/Compiler) based on the `CMAKE_BUILD_TYPE` set.

Reply:
You could leave out the `compile_options`, but I still think the `compile_definitions` are useful to have.

Reply:
Still might recommend keeping the `-O0` for debug, but up to you

Reply:
Done in https://github.com/bdaiinstitute/ros_utilities/pull/118/commits/2c0bf18ebcf5e4cb6951b6cbb02a391b3d442bb1.